av/enum.pyi

```diff
@@ -18,17 +29,17 @@ class EnumItem:
 
     def __int__(self) -> int: ...
     def __hash__(self) -> int: ...
-    def __reduce__(self): ...
-    def __eq__(self, other) -> bool: ...
-    def __ne__(self, other) -> bool: ...
+    def __reduce__(self) -> tuple[bool, ...]: ...
```

Comment:
I actually have no clue what `__reduce__` returns, or if this method is even needed at all. Just use set it to `Any`

Reply:
Should be `tuple[Callable[[str, str, str], Any], tuple[str, str, str]]`.

`enum.pyx`:
```python
    def __reduce__(self):
        return (_unpickle, (self.__class__.__module__, self.__class__.__name__, self.name))
```

```python
def _unpickle(mod_name, cls_name, item_name):
    mod = __import__(mod_name, fromlist=["."])
    cls = getattr(mod, cls_name)
    return cls[item_name]
```

Reply:
Mystery solved:

```python
>>> from av.video.frame import PictureType
>>> t = PictureType
>>> t
<class 'av.video.frame.PictureType'>
>>> t[0]
<av.video.frame.PictureType:NONE(0x0)>
>>> t[0].__reduce__()
(<cyfunction _unpickle at 0x73e2344805f0>, ('av.video.frame', 'PictureType', 'NONE'))
>>> t[0].__reduce__()[0](*t[0].__reduce__()[1])
<av.video.frame.PictureType:NONE(0x0)>
>>> type(t[0].__reduce__()[0](*t[0].__reduce__()[1]))
<class 'av.video.frame.PictureType'>
```

So it should be tuple[Callable[[str, str, str], EnumItem], tuple[str, str, str]]